t3f/ops.py

```diff
@@ -20,12 +21,20 @@ def full(tt):
   ranks = shapes.lazy_tt_ranks(tt)
   shape = shapes.lazy_shape(tt)
   raw_shape = shapes.lazy_raw_shape(tt)
+  if isinstance(tt, TensorTrainBatch):
```

Comment:
Kinda fixed. I dont want to do 2 separate functions because it will increase code duplication, but I at least gathered all batch related code into one place.